docs/source/developer/mission.rst

```diff
@@ -0,0 +1,83 @@
+Numba Mission Statement
+=======================
+
+Introduction
+------------
+
+The mission statement describes the goals and purposes of the Numba project and
+it exists to provide a clear description of these purposes and goals.  As
+such, this document provides background on Numba's users and use-cases, and
+also outlines the project's overall goals and its future direction.
```

Comment:
```suggestion
such, this document provides background on Numba's users and use-cases, and
outlines the project's overall goals.
```
the future direction is not mentioned in this document? I think this is what a "roadmap" would describe?

Reply:
Agreed